Projects/Team2/derbackAPP/TransportColV2/ProgData.cs

```diff
@@ -0,0 +1,98 @@
+﻿using ConsoleApp2;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Xml.Serialization;
+
+namespace TransportCollection
+{
+    public class TransportData
+    {
+        internal static int GetAge(int v)
+        {
+            return DateTime.Now.Year - v;
+        }
+
+        public static void TransportVoid()
+        {
+            string keyWord = "Car";
+                Car car1 = new Car(200, "X5", "BMW", "Car", 10, true, 22.277);
+                Car car2 = new Car(220, "A8", "Audi", "Car", 8, true, 35.12);
+                Car car3 = new Car(210, "A200", "Mercedes-Benz", "Car", 13, true, 24.434);
+                Car car4 = new Car(200, "Skyline", "Nissan", "Car", 11, false, 27.13);
+                Car car5 = new Car(240, "X6", "BMW", "Car", 6, true, 21.44);
+                Car car6 = new Car(200, "Omega", "Opel", "Car", 20, false, 5.4);
+                Car car7 = new Car(230, "Fusion", "Ford", "Car", 12, true, 12.3);
+                Car car8 = new Car(320, "Huricane", "Lamborgini", "Car", 3, true, 223.5);
+                Car car9 = new Car(260, "Phantom", "Rols-Royse", "Car", 5, true, 157.4);
+
+                List<Car> cars = new List<Car>();
+                cars.Add(car9);
+                cars.Add(car8);
+                cars.Add(car7);
+                cars.Add(car6);
+                cars.Add(car5);
+                cars.Add(car4);
```

Comment:
we can make this in one line cars.Add(new car(...))

or just like this List<int> numbers = new List<int>() { new car(...), new car(...), .... };